manual/cloud/README.md

```diff
@@ -1,22 +1,19 @@
-## Connecting to Apollo (Cloud)
+## Connecting to Astra (Cloud)
 
-Using the DataStax Java Driver to connect to a DataStax Apollo database is almost identical to using
+Using the DataStax Java Driver to connect to a DataStax Astra database is almost identical to using
 the driver to connect to any normal Apache Cassandra® database. The only differences are in how the
 driver is configured in an application and that you will need to obtain a `secure connect bundle`.
 
-The following is a Quick Start guide to writing a simple application that can connect to an Apollo
+The following is a Quick Start guide to writing a simple application that can connect to an Astra
 database.
 
 ### Prerequisites
 
 1. [Download][Download Maven] and [install][Install Maven] Maven.
-1. Create an Apollo database on [GCP][Create an Apollo database - GCP] or
-   [AWS][Create an Apollo database - AWS]; alternatively, have a team member provide access to their
-   Apollo database (instructions for [GCP][Access an Apollo database - GCP] and 
-   [AWS][Access an Apollo database - AWS]) to obtain database connection details.
+1. Create an Astra database on [GCP/AWS/Azure][Create an Astra database - GCP/AWS/Azure]; alternatively, have a team member provide access to their
```

Comment:
Nit: try to keep line widths below 100 characters for better readability of the source code (but it doesn't affect the rendered result, of course).

Reply:
done